rmb_client/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+use std::{str::FromStr, sync::Arc};
+
+use anyhow::{Ok, Result};
+use codec::{Decode, Encode};
+use sp_core::{Pair};
+use sp_runtime::MultiSignature;
+use substrate_api_client::Api;
+
+
+#[derive(Clone)]
+pub struct RmbClient<P>
+where
+    P: Pair,
```

Comment:
Since this `RmbClient` is a temporary solution until we move to subxt. it doesn't actually have to have generic type P, right ?

I suggest we drop this and instead make

```rust
pub struct SubstrateClient {
   api: Arc<Api<ed25519::Pair>>,
}
```